### PR TITLE
feat: account cards

### PR DESCRIPTION
**Pull request by eddiejaoude**
Ignore numbers, my test data is a bit messy

Screenshot 2023-02-11 at 11 10 22

### CODE REVIEW

File: pages/account/statistics.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ import Alert from "../../components/Alert";
 import Page from "../../components/Page";
 import PageHead from "../../components/PageHead";
 import { abbreviateNumber } from "../../services/utils/abbreviateNumbers";
+import BasicCards from "../../components/statistics/BasicCards";
 
 export async function getServerSideProps(context) {
   const session = await unstable_getServerSession(
```
Comment (krshkun) [inline pages/account/statistics.js:22]: Just as a reminder, on newer versions of next-auth this function is not unstable, so we might want to update ✅

**Review by krshkun — COMMENTED** [review]
I did not understand what exactly was changed, but I do not think I see something that breaks anything, waiting for other reviews 👍

File: pages/account/statistics.js (hunk 1 above)
Comment (eddiejaoude) [reply pages/account/statistics.js:22]: good spot, yes this has been deprecated 👍 we will need a separate PR to update all of these

**Comment by eddiejaoude** [comment]
> I did not understand what exactly was changed, but I do not think I see something that breaks anything, waiting for other reviews 👍

Thanks for reviewing 👍  the changes are the 3 boxes at the top of the page above the graph

**Pull request merged**